main.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable linebreak-style */
```

Comment:
- [ ] please as the previous reviewer asked try to remove the  eslint disable in  because it ensures to write better code so I encourage you to do it

Reply:
And the last commit clearly shows that the line highlighted was deleted.